.github/workflows/go.yml

```diff
@@ -65,7 +65,13 @@ jobs:
               with:
                   redis-version: ${{ matrix.redis }}
 
-            - name: Install client dependencies
+            - name: Install tools for Go 1.18
+              if: ${{ matrix.go == '1.18' }}
+              working-directory: ./go
+              run: make install-tools-go1.18
```

Comment:
Sure, I can do this for the workflows that don't use amazon linux. But I can't do this for the amazon linux workflow because the versions are different (see comment below as to why they are different)